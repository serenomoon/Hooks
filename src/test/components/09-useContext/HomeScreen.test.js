import React from 'react';
import { mount } from 'enzyme';
import { HomeScreen } from '../../../components/09-useContext/HomeScreen';
import { UserContext } from '../../../components/09-useContext/UserContext';


describe('Test <HomeScreen />', () => {
    
    const user = {
        user: 'Saulo',
        email: 'saulo@email.com'
    }

    const wrapper = mount(
    <UserContext.Provider value = {{
        user
    }}>
        <HomeScreen />
    </UserContext.Provider>
    );

    test('should show snapshot correctly', () => {


        expect( wrapper ).toMatchSnapshot();

     })

});
