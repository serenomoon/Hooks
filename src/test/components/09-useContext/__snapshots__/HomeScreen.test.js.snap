// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <HomeScreen /> should show snapshot correctly 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {
   "user": "Saulo",
   "email": "saulo@email.com"
}
  </pre>
</div>
`;
